src/main.js

```diff
@@ -722,15 +773,13 @@ function addEleContent(e) {
   else {
       console.warn("Node/Link NOT FOUND");
   }
-
-
 }
 
 function showContent(ele) {
-    var addressToFrame = "http://webstrates.ucsd.edu/" + ele;
-    var wrapper = document.createElement('div');
+    let addressToFrame = "http://webstrates.ucsd.edu/" + ele;
+    let wrapper = document.createElement('div');
     wrapper.setAttribute("id", "showing");
-    var toFrame = '<iframe src="'+ addressToFrame + '" style="position: absolute;left: 8px;top: 8px;width: 300px;height: 300px;"><p>ERROR: Your browser does not support iframes.</p></iframe>';
+    let toFrame = '<iframe src="'+ addressToFrame + '" style="position: absolute;right: 8px;bottom: 8px;width: 300px;height: 300px;"><p>ERROR: Your browser does not support iframes.</p></iframe>';
```

Comment:
Use JS to add element and change style in css.

Reply:
Done